test/curator.sh

```diff
@@ -437,17 +437,17 @@ EOF
     return 0
```

Comment:
Instead, have something near the beginning of the test
```
if using cronjobs ; then
    os::log::info "This test does not support curator cronjobs yet - skipping"
    exit 0
fi
```

Reply:
I was planning on getting the test to work but if I cannot figure it out until the curator image changes are merged in then i'll add that if block to the top

Reply:
ok